src/svg.select.js

```diff
@@ -11,23 +11,30 @@ function SelectHandler(el) {
       rt: [ 'width', 0 ],
       rb: [ 'width', 'height' ],
       lb: [ 0, 'height' ],
-      t: [ 'width / 2', 0 ],
-      r: [ 'width', 'height / 2' ],
-      b: [ 'width / 2', 'height' ],
-      l: [ 0, 'height / 2' ]
+      t: [ 'width', 0 ],
+      r: [ 'width', 'height' ],
+      b: [ 'width', 'height' ],
+      l: [ 0, 'height' ]
     };
 
-    // helper function to evaluate point coordinates based on settings above and an object (bbox in our case)
-    this.pointCoord = function (setting, object) {
-      return typeof setting !== 'string' ? setting : eval('object.' + setting)
+    // helper function to get point coordinates based on settings above and an object (bbox in our case)
+    this.pointCoord = function (point, index, object) {
+      var setting = this.pointsList[point][index];
+      var coord = typeof setting !== 'string' ? setting : object[setting];
+
+      // Top, bottom, right and left points are placed in the center of element width/height
+      var isXCentered = index === 0 && (point === 't' || point === 'b');
+      var isYCentered = index === 1 && (point === 'r' || point === 'l');
+      if (isXCentered || isYCentered) {
+        coord /= 2;
+      }
+      return coord
     }
 
     this.pointCoords = function (point, object) {
-      var settings = this.pointsList[point];
-
       return {
-        x: this.pointCoord(settings[0], object),
-        y: this.pointCoord(settings[1], object)
+        x: this.pointCoord(point, 0, object),
+        y: this.pointCoord(point, 1, object)
```

Comment:
I thought about a maybe easier way to do that :)

what about:

```js
var settings = this.pointsList[point];

return {
  x: this.pointCoord(settings[0], object, (point === 't' || point === 'b')),
  y: this.pointCoord(settings[1], object, (point === 'r' || point === 'l'))
}
```

And then in the pointCoord function you can simply check teh third argument. If its true, half the value :)

Reply:
@Fuzzyma sure, I'm ok with that. I already pushed the changes.
I updated commit using 'commit --amend'. I'm new in open source contributing, please let me know if I did something wrong :)